addons/mrp/mrp_view.xml

```diff
@@ -562,6 +562,52 @@
                 </search>
             </field>
         </record>
+
+        <record id="product_template_search_view_procurment" model="ir.ui.view">
```

Comment:
@tde-banana-odoo ,
Yes, This is duplicate view. but, do not need this then I remove it.